src/v/cloud_storage/remote.h

```diff
@@ -85,7 +85,8 @@ class remote {
       const s3::bucket_name& bucket,
       const remote_manifest_path& key,
       base_manifest& manifest,
-      retry_chain_node& parent);
+      retry_chain_node& parent,
+      bool supress_not_available_message = false);
```

Comment:
you may be interested in using `ss::bool_class` as a strong type for booleans.

Reply:
Noted! but after force-push I don't need it anymore